fix: 2.7 and 3.0 metadata system (#1519)
api-platform/docs: core/identifiers.md

    /**
     * {@inheritdoc}
     */

>      public function supports(string $resourceClass, array $identifiers = [], ?string $operationName = null, array $context = []): bool
>      {

Reviewer: ```suggestion
    public function supports(string $resourceClass, array $identifiers = [], ?string $operationName = null, array $context = []): bool
    {
```

        return $resourceClass === Person::class;

>      }

Reviewer: ```suggestion
    }
```

}
```

> To cover this use case, we need to `convert` the identifier to an instance of our `App\Uuid` class. This case is covered by an uri variable transformer:

Reviewer: ```suggestion
To cover this use case, we need to `transform` the identifier to an instance of our `App\Uuid` class. This case is covered by an uri variable transformer:
```

```php
<?php
namespace App\Identifier;

use ApiPlatform\Api\UriVariableTransformerInterface;
use ApiPlatform\Exception\InvalidUriVariableException;
use App\Uuid;

final class UuidUriVariableTransformer implements UriVariableTransformerInterface
{
    /**
     * Denormalizes data back into an object of the given class.
     *
     * @param mixed $value   The uri variable value to transform
     * @param array $types   The guessed type behind the uri variable
     * @param array $context Options available to the transformer
     *
     * @throws InvalidUriVariableException Occurs when the uriVariable could not be transformed
     */
     public function transform($value, array $types, array $context = []) {
        try {
            return Uuid::fromString($data);
        } catch (InvalidUuidStringException $e) {
            throw new InvalidUriVariableException($e->getMessage());
        }
     }

    /**

>      * Checks whether the given class is supported for denormalization by this normalizer.

Reviewer: ```suggestion
     * Checks whether the given identifier is supported for transformation by this transformer.
```
